### ISSUE TITLE
Update Wrought Havoc

### ISSUE THREAD

**Issue by queejie** [ionic-team/ionic-cli#2395]
**Note: for support questions, please use one of these channels:**

https://forum.ionicframework.com/
http://ionicworldwide.herokuapp.com/

**Note: for build related issues you can open up an issue on the ionic-app-scripts repo**

https://github.com/ionic-team/ionic-app-scripts

Please ensure that you are on the latest version of the CLI.
`npm view ionic@latest version`

#### Short description of the problem:
The update to ionic I was prompted for this morning left absolute chaos in its path.  I have no idea how to get back to a usable app.  The first message was something about sass not being supported on my architecture (OSX 64bit).  I ran npm install --save node_sass to get it to start working, but now I am getting Error: connect ECONNREFUSED 127.0.0.1:80.  After two hours of time wasted, I still have no idea how to proceed.

#### What behavior are you expecting?
An update that doesn't mess up everything to the point that there appears no way to go back.

**Steps to reproduce:**
1. respond Y to updating ionic-cli (and some other ionic thing)
2. try to run ionic serve
3. waste hours trying to recover

```
insert any relevant code between the above and below backticks
```

**Post the output of `ionic info` below please**
global packages:

    @ionic/cli-utils : 1.3.0
    Cordova CLI      :  You have been opted out of telemetry. To change this, run: cordova telemetry on. 7.0.1 
    Ionic CLI        : 3.3.0

local packages:

    @ionic/app-scripts              : 1.3.7
    @ionic/cli-plugin-cordova       : 1.2.1
    @ionic/cli-plugin-ionic-angular : 1.2.0
    Cordova Platforms               : You have been opted out of telemetry. To change this, run: cordova telemetry on.
    Ionic Framework                 : ionic-angular 3.3.0

System:

    Node       : v8.0.0
    OS         : macOS Sierra
    Xcode      : Xcode 8.3.2 Build version 8E2002 
    ios-deploy : 1.9.1 
    ios-sim    : 5.0.13 

**Other information:** (e.g. stacktraces, related issues, suggestions how to fix, stackoverflow links, forum links, etc)

**Comment by danielsogl** [comment]
@queejie downgrade to Node 6.10.3. Ionic is not Node8/npm5 ready

**Comment by queejie** [comment]
@danielsogl  Thank you, but that did not help with the ECONNREFUSED problem:

````
Error: connect ECONNREFUSED 127.0.0.1:80
    at Object.exports._errnoException (util.js:1018:11)
    at exports._exceptionWithHostPort (util.js:1041:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1086:14)
````

**Comment by queejie** [comment]
I have no idea how, but it is now up and running again.  I returned to Node v8, and it still works fine. It may have something to do with my using a proxy to get to server side .php calls, although nothing has changed in that regard for months.  I'm not sure what the update did, but it definitely had nothing to do with code changes on my part, since there were none.  I just started up with ionic serve this morning, it prompted me for updates, and things broke.
